src/eval.c

```diff
@@ -888,7 +888,7 @@ int ldbStartSession(client *c) {
             /* Log the creation of the child and close the listening
              * socket to make sure if the parent crashes a reset is sent
              * to the clients. */
-            serverLog(LL_NOTICE,"Redis forked for debugging eval");
+            serverLog(LL_NOTICE,"Valkey forked for debugging eval");
```

Comment:
@0del It looks good, but we are not yet sure if we want hard-coded "Valkey" in each log message.

Maybe a macro instead, like

```suggestion
            serverLog(LL_NOTICE, SERVER_TITLE " forked for debugging eval");
```

and somewhere in `server.h` we can `#define SERVER_TITLE "Valkey".

We didn't decide about this yet, so I think we can wait before we start changing all log messages.

Reply:
Thanks for explanation !!

> We didn't decide about this yet, so I think we can wait before we start changing all log messages.

Did you mean, we did not decide for definition `SERVER_TITLE` macro ?

Reply:
Right, we didn't decide about the macro and about which style we will use for log messages. There are some alternatives:

1. `"Forked for debugging eval"`
2. `"Server forked for debugging eval"`
3. `"Valkey forked for debugging eval"`
4. `SERVER_TITLE " forked for debugging eval"`

Reply:
i removed server name (`Redis`, `Valkey`) 
I think, I will pending this issue https://github.com/valkey-io/valkey/issues/207, wait for device define `marco` and pick another issue and work on it.  
Thanks for your supports

Reply:
Ah, you're already assigned to that issue! I understand now.

I did #206 and it is not yet decided exactly.